CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 - Add tuist version to the target hash computation. [#3455](https://github.com/tuist/tuist/3455) by [@danyf90](https://github.com/danyf90)
 - Fix unauthenticated cache exists responses interpreted as existing build artifact. [#3480](https://github.com/tuist/tuist/3480) by [@danyf90](https://github.com/danyf90)
 - Fix `.tuistignore` not matching relative paths correctly [#3456](https://github.com/tuist/tuist/pull/3456) by [@danyf90](https://github.com/danyf90)
+- Sanitize target names in resolved dependencies. [#3495](https://github.com/tuist/tuist/3495) by [@moritzsternemann](https://github.com/moritzsternemann)
```

Comment:
Given we haven't done a release yet, I think the other changelog entry is enough
```suggestion
```